pyensembl/locus.py

```diff
@@ -1,10 +1,10 @@
 def normalize_chromosome(c):
-    if isinstance(c, (int, long)):
+    if isinstance(c, int):
```

Comment:
Again, this changes the meaning a little bit for Python 2.7.x. I guess it's fine, since getting a `long` for a chromosome seems like an edge case but it does take us even further from dynamic typing.